README.md

```diff
@@ -1,2 +1,11 @@
 # 🚧 Trabalho em progresso.
+
 Repositório para o projeto de front-end
+
+# Como contribuir
```

Comment:
Não é uma boa prática ter mais de um header H1
```suggestion
## Como contribuir
```

fonte: https://github.com/DavidAnson/markdownlint/blob/v0.20.2/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document